lagunita/cms/templates/widgets/sock_links_extra.html

```diff
@@ -0,0 +1,19 @@
+<%page expression_filter="h" />
+<%!
+from django.conf import settings
```

Comment:
Are we using `settings`? Do we need this?

Reply:
We don't! This is a remnant from "copy-paste".